server/users.ts

```diff
@@ -308,6 +308,10 @@ type ChatQueueEntry = [string, RoomID, Connection];
 export interface UserSettings {
 	blockChallenges: boolean;
 	blockPMs: boolean | AuthLevel;
+	status: {
+		type: StatusType,
+		message: string,
+	};
```

Comment:
It seems to me like you would have less code if this was `statusType` and `statusMessage`.

Reply:
thought it would be cleaner to handle it similar to #7191, but I can move it to just statusType and statusMessage if you'd prefer.

Reply:
(also imo it's slightly cleaner to parse in /updatesettings, since it requires one "else if" as an object, not two, but that's just me)

Reply:
Actually, on reflection, i agree - i'll do this now